#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()
gen.add("grid_size", double_t, 0, "grid size", 0.2, 0.0, 2.0)
gen.add("min_indices", int_t, 0, "the minimum numnber of the indices for each grid", 0, 0, 1000)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "GridSampler"))
